CMakeLists.txt

```diff
@@ -66,7 +96,9 @@ if(NOT CMAKE_HOST_WIN32 OR CYGWIN OR MINGW)
     set(includedir  ${CMAKE_INSTALL_FULL_INCLUDEDIR})
     set(LIBS        "-lz -lm")
```

Comment:
Not touched by your patch, but here `"-lz"` can be dropped, as the .pc file already contains `Requires: zlib` (see https://github.com/randy408/libspng/pull/239)